Formula/php.rb

```diff
@@ -28,6 +29,8 @@ class Php < Formula
   depends_on "libpng"
   depends_on "libpq"
   depends_on "libsodium"
+  # libxml2 required due to https://bugs.php.net/bug.php?id=76403, can be removed once resolved
```

Comment:
This should be wrapped to 80 characters, as much as I appreciate that's annoying 🙃.